test_cases/general/map/of/class_ref/go/test_generate/to_jsonable_test.go

```diff
@@ -0,0 +1,5 @@
+package somegraph
+
+// File automatically generated by mapry. DO NOT EDIT OR APPEND!
+
+// File automatically generated by mapry. DO NOT EDIT OR APPEND!
```

Comment:
@mristin is this correct? Why did the message appear twice?

Reply:
It is correct. I did not check if there is no body to be generated. Let me create an issue -- this should be actually fixed for cosmetic reasons. I'll submit a fix in a separate PR.